sql-db/hibernate-fulltext-search/src/main/java/io/quarkus/qe/hibernate/search/FruitResource.java

```diff
@@ -6,21 +6,22 @@
 import javax.inject.Inject;
 import javax.persistence.EntityManager;
 import javax.transaction.Transactional;
-import javax.validation.constraints.NotNull;
```

Comment:
Why is `@NotNull` used? Some limitation of RESTEasy Reactive?

Reply:
Was removed from the previous implementation, in this scenario is not adding any value.

Reply:
ok, thanks